pkgs/development/python-modules/graphite_beacon/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, tornado, pyyaml, funcparserlib
+}:
+buildPythonPackage rec {
+  pname = "graphite_beacon";
+  version = "0.27.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "03bp4wyfn3xhcqyvs5hnk1n87m4smsmm1p7qp459m7j8hwpbq2ks";
+  };
+
+  propagatedBuildInputs = [ tornado pyyaml funcparserlib ];
+
+  patchPhase = "> requirements.txt";
```

Comment:
Why is this needed?

Reply:
This was in the [original package](https://github.com/NixOS/nixpkgs/pull/44276/files#diff-2afbba83e1b14f9e0c304c769831a0e2L15670) and without it the build fails at the dependency checking phase. I do feel a bit uneasy about just ignoring dependency version ranges.

Reply:
How about using
```
substituteInPlace requirements.txt "==" ">="
```
instead?
Also, this should be put into `postPatch`.

Reply:
Thanks, that's better indeed.